modules/contact/test/tests/multiple_contact_pairs/tests

```diff
@@ -32,8 +32,8 @@
     cli_args = "Postprocessors/active=''"
     map = False
     partial = True
-    requirement = 'The system shall set up mortar interactions between multiple surface pairs '
-                  'using the Contact action with distinct input blocks.'
+    requirement = 'The system shall set up mortar interactions between multiple surface pairs using '
+                  'the Contact action with distinct input blocks.'
```

Comment:
```suggestion
                  'a contact action syntax with distinct input blocks.'
```

Reply:
Should this be "_a_ contact action syntax" or simply "contact action syntax"? Anyway, making the change.

Reply:
yea I think either is good